README.md

```diff
@@ -124,13 +126,19 @@ javaLibrary {
 
 </details>
 
+## Get Started
+
+See the [Getting Started Guides](./docs/getting-started/README.md).
+
 ## Prototypes
 
 Here are the experimental prototypes
-currently available for initial review.
-They are not ready for evaluation at the moment:
+currently available for initial review and evaluation:
 
-- [Unified Prototype](./unified-prototype/README.md) - prototypes of plugins for JVM, Android, and KMP projects built using "unified" plugins that all utilize a similar model and implemented using the Declarative DSL
+- [Declarative Gradle Prototype](./unified-prototype/README.md) - prototypes of plugins for JVM, Android, and KMP projects built using "unified" plugins that all utilize a similar model and implemented using the Declarative DSL
```

Comment:
💭 Should this explicitly mention non-KMP Kotlin as well here?

Reply:
```suggestion
- [Declarative Gradle Prototype](./unified-prototype/README.md) - prototypes of plugins for JVM, Android, Kotlin and KMP projects built using "unified" plugins that all utilize a similar model and implemented using the Declarative DSL
```